reconcile/aws_iam_keys.py

```diff
@@ -48,7 +48,7 @@ def init_tf_working_dirs(
     accounts: Iterable[dict[str, Any]],
     thread_pool_size: int,
     settings: Mapping[str, Any],
-) -> dict[str, str]:
+) -> Mapping[str, str]:
```

Comment:
What is the reason for this change? 
IINW, the general recommendation was using abstract types for parameters and concrete types for return types. 

Just commenting here but there are other ocurrences with the same change (dict -> Mapping/MutableMapping)

Reply:
+1, for return type, use concrete or generic types

Reply:
I changed the return types of mapping in the last commits but in some functions like dump I was receiving a parameter modifying it and returning it. If I wanted to receive a mapping I was required to return a mapping.